cristin-organization/src/main/java/no/unit/nva/cristin/organization/CristinOrgApiClientVersion2.java

```diff
@@ -0,0 +1,66 @@
+package no.unit.nva.cristin.organization;
```

Comment:
have put models in own package. will probably do the same with clients when i restructure the package tree. Since it now has more classes there is a need to rework the package tree